documents/extension/funding/service.go

```diff
@@ -95,17 +105,17 @@ func getFundingsLatestIDX(model documents.Model) (idx *documents.Int256, err err
 
 }
 
-func incrementFundingAttrIDX(model documents.Model) (attr documents.Attribute, err error) {
-	key, err := documents.AttrKeyFromLabel(fundingLabel)
+func incrementArrayAttrIDX(model documents.Model, arrayLabel string) (attr documents.Attribute, err error) {
```

Comment:
I think the previous function name makes more sense IMO instead making this array specific

Reply:
I am re-using the same method for the signatureArray